### PR TITLE
Set jpath via flag

### LINKED ISSUE #24: Add support of custom Jsonnet library paths
Configuration variable **JsonnetPath** is present in [pkg/grizzly/config.go](https://github.com/grafana/grizzly/blob/master/pkg/grizzly/config.go#L11) but not used. It is really desirable to have the possibility of defining a specific path for Jsonnet libraries.
```golang
// Config provides configuration to `grafana-dash`
type Config struct {
	GrafanaURL  string
	JsonnetPath string
}
```
Also it will be a plus to have the possibility of override the path with the environment variable **JSONNET_PATH**

### PR DESCRIPTION
**Pull request by mavimo**
Closes #24

### CODE REVIEW

**Comment by mavimo** [comment]
@malcolmholmes ready to be reviewed :)

**Comment by malcolmholmes** [comment]
Thanks for this. This looks (mostly) fine. There's a few things (e.g. whitespace change and `log.SetFlags`) that aren't pertinent to this PR - could we remove them?

My only concern really is that the default jsonnet path values are now cloned to multiple places. Perhaps they could be set within `config` as `DefaultJsonnetPath` and consumed from there in your `--jpath` flag definitions?

**Comment by mavimo** [comment]
@malcolmholmes the reason why is not included in `grizzly.Config` is that this value is set from the command only when is needed (when we process jsonnet file, as happen the file to process), I'm not sure that having this config in the global config is a good idea because:

 - We expose the flag also on command that don't need it (allowing to set config that are not relevant, as should happen in `list` o `providers` commands)
 - We add a property in a config that should change time by time with value that should be nil when not set and we need to understand if in the current context the config should have (or not) the jpath

About log changes you're right, I'm going to remove and open a new PR for that.

**Comment by mavimo** [comment]
@malcolmholmes PR rebased, I would like to squash all commits in a single one, it's fine for you?

**Comment by malcolmholmes** [comment]
Can we somehow define the default jpath in only one place? Other than that, Happy to merge/squash.

**Comment by mavimo** [comment]
@malcolmholmes
> Can we somehow define the default jpath in only one place? 
ok, I'll ping you once done

**Comment by mavimo** [comment]
@malcolmholmes done

File: cmd/grr/workflow.go [hunk 1]
```diff
@@ -210,3 +223,7 @@ func providersCmd(config grizzly.Config) *cli.Command {
 	}
 	return cmd
 }
+
+func getJsonnetFolders() []string {
```
Comment (malcolmholmes) [inline cmd/grr/workflow.go:227]: Can we make it this:
```suggestion
func getDefaultJsonnetFolders() []string {
```

Then I think we can merge this.

Comment (mavimo) [reply cmd/grr/workflow.go]: Done :)

Comment (mavimo) [reply cmd/grr/workflow.go]: @malcolmholmes Done :)

**Comment by mavimo** [comment]
@malcolmholmes do you have time to check if something is missing (no pressure, just to know if there is something missing from my side)

**Review by malcolmholmes — APPROVED** [review]

**Pull request merged**